tslint.json

```diff
@@ -1,7 +1,10 @@
 {
   "extends": "tslint:latest", // Widget dev rules base
   "rules": {
-    // Development rules mendix web modeler team
+    "interface-name": [
```

Comment:
Please move the custom widget settings, to the end of the tslint.json
The first set are the Mendix rules